ClemBot.Bot/bot/cogs/choose_cog.py

```diff
@@ -0,0 +1,84 @@
+import re
+import typing as t
+from random import choice
+
+import discord
+import discord.ext.commands as commands
+
+import bot.extensions as ext
+from bot.clem_bot import ClemBot
+from bot.consts import Colors
+from bot.utils.logging_utils import get_logger
+
+log = get_logger(__name__)
+
+
+class ChooseCog(commands.Cog):
+    def __init__(self, bot: ClemBot) -> None:
+        self.bot = bot
+
+    @ext.command()
+    @ext.long_help("Given a comma separated list, this command will choose between them randomly")
+    @ext.short_help("Chooses an item from the given list randomly")
+    @ext.example(
+        (
+            "choose a,b,c,d,e",
+            "choose a, b, c, d, e",
+        )
+    )
+    async def choose(self, ctx: ext.ClemBotCtx, *, choices_list: str) -> None:
+        choices_list = choices_list
+
+        error_title = ""
+        error_message = ""
+
+        # split string on comma to get separate options
+        options = choices_list.split(",")
+
+        # remove leading/trailing spaces to ensure no whitespace from original formatting
+        options = [x.lstrip() for x in options]
+
+        # check that there are at least 2 arguments
+        if len(options) < 2:
+            error_title = "Invalid argument(s)"
+            error_message = f"Are you sure you used the correct format? See `{await self.bot.current_prefix(ctx)}help choose` for info."
+
+        # check whether any spaces remain in the options
+        elif any(bool(re.search(r"\s", ele)) for ele in options):
```

Comment:
We'd also like for the user to be able to use multi-word options as well, for example:
`!choose a dog, a house, a fish, a boat`

I believe this condition would be triggered given the prior input, so this can be removed.